--- origsrc/gedit-code-assistance-3.14.1/indentbackends/c/Makefile.am	2014-04-17 14:30:44.000000000 -0500
+++ src/gedit-code-assistance-3.14.1/indentbackends/c/Makefile.am	2015-01-09 15:12:24.942168100 -0600
@@ -4,8 +4,8 @@ indentbackends_c_libgcaindentbackendc_la
 	indentbackends/c/gca-c-backend.vala
 
 indentbackends_c_libgcaindentbackendc_la_CFLAGS = $(GEDIT_CFLAGS) -I$(top_srcdir)/src
-indentbackends_c_libgcaindentbackendc_la_LIBADD = $(GEDIT_LIBS)
-indentbackends_c_libgcaindentbackendc_la_LDFLAGS = -module -avoid-version
+indentbackends_c_libgcaindentbackendc_la_LIBADD = src/libcodeassistance.la $(GEDIT_LIBS)
+indentbackends_c_libgcaindentbackendc_la_LDFLAGS = -module -avoid-version -no-undefined
 
 indentbackends_c_libgcaindentbackendc_la_VALAFLAGS =		\
 	--vapidir $(builddir)/vapi				\
--- origsrc/gedit-code-assistance-3.14.1/src/Makefile.am	2014-10-31 11:37:56.000000000 -0500
+++ src/gedit-code-assistance-3.14.1/src/Makefile.am	2015-01-09 15:08:31.335003700 -0600
@@ -60,7 +60,8 @@ src_libcodeassistance_la_VALAFLAGS += --
 endif
 
 src_libcodeassistance_la_CFLAGS = $(GEDIT_CFLAGS) -I$(top_srcdir) -w
-src_libcodeassistance_la_LDFLAGS = -module -shared -avoid-version $(GEDIT_LIBS)
+src_libcodeassistance_la_LDFLAGS = -module -shared -avoid-version -no-undefined
+src_libcodeassistance_la_LIBADD = $(GEDIT_LIBS)
 
 vapidir = $(datadir)/vala/vapi
 dist_vapi_DATA = 	\
